client/src/components/OrderCard/OrderCard.tsx

```diff
@@ -0,0 +1,125 @@
+import cn from 'classnames';
+import React, { useState } from 'react';
+import { OrderDto } from '../../dtos/orders/Order.dto';
+import { OrderItemDto } from '../../dtos/orders/OrderItem.dto';
+import { useAppDispatch } from '../../hooks/redux';
+import { cancelOrder } from '../../stateManager/actionCreators/orders';
+import { CustomButton } from '../common/CustomButton/CustomButton';
+import { OrderedProductCard } from '../OrderedProductCard/OrderedProductCard';
+import styles from './OrderCard.module.scss';
+
+interface IProps {
+    order: OrderDto;
+}
+
+export const OrderCard: React.FC<IProps> = ({ order }) => {
+    const dispatch = useAppDispatch();
+    const [isActiveOrder, setIsActiveOrder] = useState(false);
+
+    const renderCancelButton = () => {
+        if (order.status.value !== 'processing') {
+            return null;
+        }
+
+        return (
+            <CustomButton
+                startColor='red'
+                onClick={(e) => {
+                    e.preventDefault();
+                    e.stopPropagation();
+                    dispatch(cancelOrder(order.id));
+                }}
+            >
+                Cancel
+            </CustomButton>
+        );
+    };
+
+    const renderOrderItemsString = () => {
+        return order.orderItems
+            .reduce(
+                (acc, orderItem) =>
+                    acc +
+                    orderItem.product.name +
+                    ': ' +
+                    orderItem.quantity +
+                    ', ',
+                '',
+            )
+            .slice(0, -2);
+    };
+
+    const renderItem = (text: string) => {
+        return <div className={styles.item}>{text}</div>;
+    };
+
+    const renderFullOrderItem = (title: string, text: string) => {
```

Comment:
Proposition: rename all occurencies of OrderItem to OrderProperty to remove confusion between this and actual OrderItem (as entity)